### PR TITLE
Remove mention of things you can no longer do

### LINKED ISSUE #34128: Document describes functions that are no longer possible
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/managing-issues-and-pull-requests/using-github-actions-for-project-management

### What part(s) of the article would you like to see updated?

Mentions of things you can only do on Projects V1 boards should be removed. 

Or at least clarified to say that they only apply to V1 boards.

As Projects v1 is being turned off in ~24 days and this functionality will be completely lost, I feel removing it is most appropriate.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by max-allan-cgr**
### Why:

Closes:  https://github.com/github/docs/issues/34128

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Removed mention of some features only possible on Projects V1. Because v1 is ~24 days away from being completely disabled.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@max-allan-cgr Thanks so much for opening a PR! I'll get this triaged for review ✨

**Comment by jester0251** [comment]
Please hack me . I was Just experimenting 2 dans. Now u guys think im gay
terrorist or worst. U the genius. The otherbguy dhould see im notif he
spubon me . U got my ip never i nsulted like That.pff.  yes i shouldnt
attack. But i feit smart. Sorry 4 That

**Comment by jester0251** [comment]
Please hack me u see .

File: content/actions/managing-issues-and-pull-requests/using-github-actions-for-project-management.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ shortTitle: Actions for project management
 ---
 
 
-You can use {% data variables.product.prodname_actions %} to automate your project management tasks by creating workflows. Each workflow contains a series of tasks that are performed automatically every time the workflow runs. For example, you can create a workflow that runs every time an issue is created to add a label, leave a comment, and move the issue onto a {% data variables.projects.projects_v1_board %}.
+You can use {% data variables.product.prodname_actions %} to automate your project management tasks by creating workflows. Each workflow contains a series of tasks that are performed automatically every time the workflow runs. For example, you can create a workflow that runs every time an issue is created to add a label and leave a comment.
```
Comment (stevecat) [inline content/actions/managing-issues-and-pull-requests/using-github-actions-for-project-management.md:17]:
```suggestion
You can use {% data variables.product.prodname_actions %} to automate your project management tasks by creating workflows. Each workflow contains a series of tasks that are performed automatically every time the workflow runs. For example, you can create a workflow that runs every time an issue is created to add a label{% ifversion projects-v1 %}, move the issue onto a {% data variables.projects.projects_v1_board %},{% endif %} and leave a comment.
```

File: content/actions/managing-issues-and-pull-requests/using-github-actions-for-project-management.md [hunk 2]
```diff
@@ -24,20 +24,17 @@ Many workflow triggers are useful for automating project management.
...
 * A comment is added to an issue.
-* A project card is created or moved.
 * A scheduled time.
 
 For a full list of events that can trigger workflows, see "[AUTOTITLE](/actions/using-workflows/events-that-trigger-workflows)."
 
 ## What can workflows do?
 
-Workflows can do many things, such as commenting on an issue, adding or removing labels, moving cards on {% data variables.projects.projects_v1_boards %}, and opening issues.
+Workflows can do many things, such as commenting on an issue, adding or removing labels, and opening issues.
```
Comment (stevecat) [inline content/actions/managing-issues-and-pull-requests/using-github-actions-for-project-management.md:33]:
```suggestion
Workflows can do many things, such as commenting on an issue, adding or removing labels, {% ifversion projects-v1 %}moving cards on {% data variables.projects.projects_v1_boards %}, {% endif %}and opening issues.
```

File: content/actions/managing-issues-and-pull-requests/using-github-actions-for-project-management.md [hunk 3]
```diff
@@ -24,20 +24,17 @@ Many workflow triggers are useful for automating project management.
...
 
 You can learn about using {% data variables.product.prodname_actions %} for project management by following these tutorials, which include example workflows that you can adapt to meet your needs.
 
 * "[AUTOTITLE](/actions/managing-issues-and-pull-requests/adding-labels-to-issues)"
-* "[AUTOTITLE](/actions/managing-issues-and-pull-requests/removing-a-label-when-a-card-is-added-to-a-project-board-column)"
-* "[AUTOTITLE](/actions/managing-issues-and-pull-requests/moving-assigned-issues-on-project-boards)"
 * "[AUTOTITLE](/actions/managing-issues-and-pull-requests/commenting-on-an-issue-when-a-label-is-added)"
```
Comment (stevecat) [inline content/actions/managing-issues-and-pull-requests/using-github-actions-for-project-management.md:38]:
```suggestion
{%- ifversion projects-v1 %}* "[AUTOTITLE](/actions/managing-issues-and-pull-requests/removing-a-label-when-a-card-is-added-to-a-project-board-column)"
* "[AUTOTITLE](/actions/managing-issues-and-pull-requests/moving-assigned-issues-on-project-boards)"{% endif %}
* "[AUTOTITLE](/actions/managing-issues-and-pull-requests/commenting-on-an-issue-when-a-label-is-added)"
```

**Review by guntrip — COMMENTED** [review]
Thanks for opening this, @max-allan-cgr! ✨ 

I've just made some suggested changes to use the `projects-v1` feature flag. When Projects V1 is removed, it will still be available on GitHub Enterprise Server for a while, and we'll be updating this feature flag to gracefully remove the documentation.

I'll apply those changes now, check everything is working, and then get this merged!

**Review by guntrip — APPROVED** [review]
I managed to introduce a conflict there which I had to fix but we're all set now. I'll get this merged.

Thanks again for your contribution! ✨

**Pull request merged**